rfc6962-bis.xml

```diff
@@ -1130,6 +1130,43 @@ entry specified by <spanx style="verb">start</spanx>.
         <t>See <xref target="verify_hash" /> for an outline of how to use a complete list of <spanx style="verb">leaf_input</spanx> entries to verify the <spanx style="verb">root_hash</spanx>.</t>
       </section>
 
+      <section title="Get Entry Number for SCT">
+	<t>
+	  GET https://&lt;log server&gt;/ct/v2/get-entry-for-sct
+	</t>
+	<t>
+          <list style="hanging">
+            <t hangText="Inputs:">
+              <list style="hanging">
+                <t hangText="sct:">
+                  A base64 encoded <spanx style="verb">TransItem</spanx> of type <spanx style="verb">x509_sct_v2</spanx> or <spanx style="verb">precert_sct_v2</spanx> allegedly from this log.
```

Comment:
s/from this log/signed by this log's private key/

"from this log" is too permissive.  A cert containing embedded SCTs from Log A and Log B might be logged to Log C.  If a client calls get-entries to fetch the corresponding log entry from Log C, then it will have obtained SCTs "from this log" (Log C) signed by the private keys of Logs A and B.